sdk/cognitivelanguage/ai-language-text/src/textAnalysisClient.ts

```diff
@@ -654,8 +654,11 @@ export class TextAnalysisClient {
     }
 
     if (isStringArray(documents)) {
-      const language = (languageOrOptions as string) || this.defaultLanguage;
-      realInputs = convertToTextDocumentInput(documents, language);
+      const languageHint =
```

Comment:
If `options.defaultLanguage === undefined` is true, we use the default language in client-level which make sense to me. What is the reason to be `undefined` if `options.defaultLanguage` has value?

Reply:
If `options.defaultLanguage` is set, we shouldn't send a language hint to the service. Notice that `languageHint` is used to populate the `language` field in each document object in the input.